docs/nidigital/class.rst

```diff
@@ -1557,6 +1557,66 @@ export_signal
 
             :type output_terminal: str
 
+fetch_capture_waveform
+----------------------
+
+    .. py:currentmodule:: nidigital.Session
+
+    .. py:method:: fetch_capture_waveform(site_list, waveform_name, samples_to_read, timeout)
+
+            Returns a list of named tuples (Waveform) that <FILL IN THE BLANK HERE>
+
+            Fields in Waveform:
+
+            - **site** (int)
+            - **data** (array.array of int)
+
+            
+
+
+
+            :param site_list:
+
+
+                
+
+
+            :type site_list: str
+            :param waveform_name:
+
+
+                
+
+
+            :type waveform_name: str
+            :param samples_to_read:
+
+
+                
+
+
+            :type samples_to_read: int
+            :param timeout:
+
+
+                
+
+
+            :type timeout: float or datetime.timedelta
+
+            :rtype: list of Waveform
+            :return:
+
+
+                    List of named tuples with fields:
```

Comment:
Is the order important?
Maybe it would be nicer for clients to return a dictionary of site -> data?

Reply:
@sbethur?

Reply:
See answer above.

TL;DR - dict is better.